io/include/pcl/io/hdl_grabber.h

```diff
@@ -214,6 +230,29 @@ namespace pcl
           double cosVertOffsetCorrection;
       };
 
+      void
+      fireCurrentSweep ();
```

Comment:
Would you mind adding blank lines between functions to be consistent with the style? (Also below.)